draft-ietf-quic-transport.md

```diff
@@ -2678,6 +2690,29 @@ advised to bundle as few streams as necessary in outgoing packets without losing
 transmission efficiency to underfilled packets.
 
 
+## Extension Frames (#extension-frames)
```

Comment:
The build is complaining because it should be `{#extension-frames}` (or an omitted anchor).